jobs/Frontend/movieDatabase/src/utils/moviesService.ts

```diff
@@ -0,0 +1,26 @@
+import {MovieDetails, MovieListItem} from "../types/movies";
+import {Keychain} from "../../api";
+
+const API_KEY = Keychain.movieApi;
+
+export async function getMoviesByTitle(searchTitle: string): Promise<MovieListItem[]> {
+    try {
+        const movies = await fetch(`https://api.themoviedb.org/3/search/movie?api_key=${API_KEY}&language=en-US&page=1&include_adult=false&query=${searchTitle}`)
+                           .then(response => response.json())
+                           .then(data => {return data.results})

```

Comment:
What about spread syntax?
```suggestion
                           .then(({ results }) => results)
```
Or at least:
```suggestion
                           .then(data => data.results)
```

Reply:
Actually, if you want to look this up, it's called 'destructuring'.